### PR TITLE
Design Review Landing Page Part 2

### PR DESCRIPTION
**Pull request by naufalihsank**
### Changes
1. News Section (Fix styling)
2. More Info Section (Fix styling)
3. Refactor `InfographicCarousel` component
### Task
[[Web S12][Landing Page] Design Review - News, More Info section](https://trello.com/c/FyRTfVp7/1000-web-s12landing-page-design-review-news-more-info-section)
### Evidence
title: Design review landing page part 2
project: Pikobar Website
participants: @naufalihsank @maulanayuseph @adzharamrullah @Ibwedagama

### CODE REVIEW

**Review by maulanayuseph — APPROVED** [review]

File: components/Base/ActionCard/index.vue [hunk 1]
```diff
@@ -47,8 +47,9 @@ export default {
 <style lang="scss" scoped>
 .action-card {
   @apply p-4 flex flex-col flex-no-wrap
-  rounded-md
-  bg-gray-100;
+  rounded-md;
+
+  background-color: #fafafa;
```
Comment (adzharamrullah) [inline components/Base/ActionCard/index.vue:52]: you can using arbitary value on tailwindcss (Arbitary Value)[https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values]

File: components/Base/ContentCard/index.vue [hunk 2]
```diff
@@ -123,6 +123,8 @@ export default {
   &__header {
     @apply text-sm text-brand-green-darker
     font-bold;
+
+    font-family: 'Roboto', sans-serif;
```
Comment (adzharamrullah) [inline components/Base/ContentCard/index.vue:127]: you can define font family in config tailwindcss and call every place you need

File: components/Base/ActionCard/index.vue (hunk 1 above)
Comment (naufalihsank) [reply components/Base/ActionCard/index.vue:52]: @adzharamrullah this tailwind version still `1.9.6`, so i think i can't use that feature yet

**Review by Ibwedagama — APPROVED** [review]

File: components/Base/ContentCard/index.vue (hunk 2 above)
Comment (naufalihsank) [reply components/Base/ContentCard/index.vue]: thanks! done in [here](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/446/commits/e86c4a160c21a701bc153ae68efaffa5539c4126)

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**